templates/products/rdm.html

```diff
@@ -130,6 +130,7 @@ <h2 class="section-title">Brought to you by</h2>
         <a href="https://www.uni-muenster.de/en/"><img src="{{ '/static/img/logo-uni-munster.png'|asseturl }}" title="University of Münster"></a>
         <a href="https://uni-tuebingen.de/en/"><img src="{{ '/static/img/logo-uni-tubingen.png'|asseturl }}" title="University of Tübingen"></a>
         <a href="https://www.wacren.net"><img src="{{ '/static/img/logo-wacren.png'|asseturl }}" title="West and Central African Research and Education Network"></a>
+        <a href="https://www.kth.se"><img src="{{ '/static/img/logo-kth.svg'|asseturl }}" title="KTH Royal Institute of Technology"></a>
```

Comment:
Opted for 4 items per row; unclear on alphabetical order. Feedback welcome. Thanks!